dimagi/data/sectors/sector/sub_sectors/humanitarian_response.py

```diff
@@ -12,7 +12,7 @@
         "the help of mobile data collection apps."
     ),
     template="data/sectors/content/humanitarian_response.html",
-    icon="svg/commcare/icon/agriculture.html",
+    icon="svg/commcare/icon/agricultur.html",
```

Comment:
is there are reason the spelling changed here?

Reply:
No

Actually we need to delete this file as in the process of code cleaning for the sector pages. I will do it once we are done with all the sector pages as of now I'm changing back.